src/Database/PostgreSQL/PQTypes/Model/Trigger.hs

```diff
@@ -245,15 +211,15 @@ getDBTriggers tableName = do
     sqlWhereEq "c.relname" $ unRawSQL tableName
   fetchMany getTrigger
   where
-    getTrigger :: (String, Int16, Bool, Bool, String, String, String, String) -> Trigger
-    getTrigger (tgname, tgtype, tgdeferrable, tginitdeferrable, triggerdef, proname, prosrc, tblName) =
+    getTrigger :: (String, Int16, Bool, Bool, String, String, String) -> Trigger
+    getTrigger (tgname, tgtype, tgdeferrable, tginitdeferrable, triggerdef, prosrc, tblName) =
       Trigger { triggerTable = tableName'
               , triggerName = triggerBaseName (unsafeSQL tgname) tableName'
               , triggerEvents = trgEvents
               , triggerDeferrable = tgdeferrable
               , triggerInitiallyDeferred = tginitdeferrable
               , triggerWhen = tgrWhen
-              , triggerFunction = TriggerFunction (unsafeSQL proname) (unsafeSQL prosrc)
```

Comment:
Hmm, this no longer checks that name of the trigger function in the DB matches the one generated by the library.

I don't see an obvious way to do that though and we can always add it later as it seems to be not a big deal.

Reply:
Addressed.